devito/symbolics/extended_dtypes.py

```diff
@@ -15,6 +18,62 @@
 }
 
 
+class NoDeclStruct(ctypes.Structure):
+    """A ctypes.Structure that does not generate a struct definition"""
```

Comment:
can you add a line explaining why it's needed ? basically for foreign types, but yeah, should be mentioned I think